lib/base58.dart

```diff
@@ -88,7 +88,7 @@ class Base58Decoder extends Converter<String, List<int>> {
     if (input.length == 0) return new Uint8List(0);
 
     // generate base 58 index list from input string
-    List<int> input58 = new List(input.length);
+    var input58 = List<int>.filled(input.length, 0);
```

Comment:
@stevenroose @haarts this is the only spot i'm not 💯 on if I did correctly. We cannot use `List()` in null safety and must initialize it with values unless we want to use `List<int?>` but I don't think that's what we want.

Is it ok to initialize the list with 0s?

Reply:
Yeah I think it is. Why is it a `var` though?